pkg/nsx/services/common/policy_tree.go

```diff
@@ -471,9 +532,38 @@ func (b *PolicyTreeBuilder[T]) UpdateMultipleResourcesOnNSX(objects []T, nsxClie
 		}
 		if err = nsxClient.OrgRootClient.Patch(*orgRoot, &enforceRevisionCheckParam); err != nil {
 			err = util.TransNSXApiError(err)
-			log.Error(err, "Failed to delete multiple resources on NSX with HAPI", "resourceType", b.leafType)
+			// Log failure for each resource
+			for _, obj := range objects {
+				resID := b.idGetter(obj)
+				resName := getNSXResourceName(obj)
+				resPath := b.pathGetter(obj)
+				var id, name, path string
+				if resID != nil {
+					id = *resID
+				}
+				if resName != nil {
+					name = *resName
+				}
+				if resPath != nil {
+					path = *resPath
+				}
+				log.Error(err, "Failed to delete resource", "resourceType", b.leafType, "resourceID", id, "resourceName", name, "resourcePath", path)
```

Comment:
As we only have a single error for this batch delete and we have printed all the objects in line 523 for `Attempting to delete resource`, do we need to print all the objects again here?
Maybe print the error once is enough and we know it applies to all the objects we print in 523.